[DOC]fix typo in hcfs.go

**Provide a link to that doc page:**
https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ddc/goosefs/hcfs.go
**What is the defect and your suggestions on improvement:**
hcfs.go:37:34: corrected "Compatiable" to "Compatible"

--- a/pkg/ddc/goosefs/hcfs.go
+++ b/pkg/ddc/goosefs/hcfs.go
@@ -34,7 +34,7 @@ func (e *GooseFSEngine) GetHCFSStatus() (status *datav1alpha1.HCFSStatus, err er
 
 	version, err := e.queryCompatibleUFSVersion()
 	if err != nil {
-		e.Log.Error(err, "Failed to get Compatiable Endpoint")
+		e.Log.Error(err, "Failed to get Compatible Endpoint")
 		return status, err
 	}
 

